php-nginx/install_extensions.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class InstallExtensions
+{
+    const AVAILABLE_EXTENSIONS = [
+        'bcmath',
+        'calendar',
+        'exif',
+        'ftp',
+        'gd',
+        'gettext',
+        'intl',
+        'mbstring',
+        'mysql',
+        'pcntl',
+        'shmop',
+        'soap',
+        'sqlite3',
+        'pdo_sqlite',
+        'xmlrpc',
+        'xsl',
+        ## Debian package extensions below
+        'cassandra',
+        'ev',
+        'event',
+        'grpc',
+        'imagick',
+        'mailparse',
+        'memcache',
```

Comment:
Yep, I'll add a blacklist instead of maintaining a huge whitelist. Phalcon also isn't supported on 7.1 yet.